### PR TITLE
Update spline calculations

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Calculating....

### CODE REVIEW

**Review by zanderso — APPROVED** [review]

File: packages/flutter_tools/test/general.shard/features_test.dart [hunk 1]
```diff
@@ -78,7 +78,7 @@ void main() {
     }));
 
     test('flutter web help string', () {
-      expect(flutterWebFeature.generateHelpMessage(), 'Enable or disable Flutter for web. This setting will take effect on the master and dev channels.');
+      expect(flutterWebFeature.generateHelpMessage(), 'Enable or disable Flutter for web. This setting will take effect on the master, dev, and beta channels.');
```
Comment (Zazo032) [inline packages/flutter_tools/test/general.shard/features_test.dart]: nit: master, dev and beta

Comment (jonahwilliams) [reply packages/flutter_tools/test/general.shard/features_test.dart]: I will defend the Oxford comma until my dying breath

Comment (zanderso) [reply packages/flutter_tools/test/general.shard/features_test.dart]: I agree with @jonahwilliams 

https://www.youtube.com/watch?v=W_WWx_ndVUA

Comment (Zazo032) [reply packages/flutter_tools/test/general.shard/features_test.dart]: Nevermind I didn't know this is something almost mandatory in American English, lesson learned

**Pull request merged**